src/pages/[language]/blog/[slug]/index.vue

```diff
@@ -259,11 +263,6 @@
     margin-bottom: var(--spacing-large);
   }
 
-  .page-blog-post-list__image .image-with-description__description {
-    margin-left: 0;
-    margin-right: 0;
-  }
-
```

Comment:
These, and a few other tweaks are gone. Honestly it isn't a big difference, and I wanted to keep it generic.

Reply:
I think it even looks a bit cleaner, with the flow on the right being consistent